packages/conf-openimageio/conf-openimageio.1/files/test.cpp

```diff
@@ -0,0 +1,5 @@
+#include <OpenImageIO/version.h>
+
+#if OIIO_VERSION_MAJOR >= 2
```

Comment:
Should this be `OIIO_VERSION_MAJOR < 2`?

Reply:
Indeed. My bad. Fixed.